extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -362,8 +365,8 @@ html, body {
 }
 
 .conversation-skin-right-card-container {
-  margin-left: 12px;
-  width: 1040px;
+  margin-left: 6px;
+  width: calc(100% - 380px);
```

Comment:
Do we need -webkit, -moz prefixes?

https://developer.mozilla.org/en-US/docs/Web/CSS/calc